styles/style.js

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
```

Comment:
destruct this file on common styles and styles of some Component

Reply:
@DarkProDen 
i don't understand it. Why?